doc/dev/crimson/crimson.rst

```diff
@@ -64,6 +65,57 @@ The last causes pools to be created by default with the ``crimson`` flag.
 Crimson pools are restricted to operations supported by Crimson.
 ``Crimson-osd`` won't instantiate PGs from non-Crimson pools.
 
+Configure Crimson with Bluestore
+================================
+
+Deploying Crimson with Bluestore as the back end requires setting
+several configuration values. The required options are
```

Comment:
requires setting `crimson_seastar_cpu_cores`` and ``crimson_alien_thread_cpu_cores``

No point in mentioning "several configuration values" without specifically listing them.

Reply:
I'll add these in now.

Reply:
I altered this paragraph so that it is easier to understand. Then I decided that I would wait until all the technical matters raised by @athanatos have been dealt with before I made any more alterations to the English of this section. I apologize in advance if I've stepped on toes here.